objects/tag.c

```diff
@@ -139,7 +139,6 @@ tag_view(lua_State *L, int udx, bool view)
     {
         tag->selected = view;
         banning_need_update();
-        ewmh_update_net_current_desktop();
```

Comment:
Yeah...you are right, i've confused myself it seems - what i've said would stand only for old version of the solution, not the current. 

Do we really need both unfocus and untagged? 

Regarding property::selected, i'll have to revert to the direct call to ewmh_update_current_desktop(NULL) in tag.c, since tag class is not accessible outside of tag.c